mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -55,6 +55,28 @@ static void strToTokens(const std::string &arguments,
     }
   }
 }
+
+// In multi-threaded context, static intialization is guaranteed to
+// be thread safe and atomic. With this guarantee, we are protected
```

Comment:
I do not think there is atomicity guarantee...

Reply:
Will remove, somehow I was associating the static initialization with `test-and-set` instruction which is clearly not the right analogy.